src/api/app/views/webui2/webui/request/_sourcediff_tab.html.haml

```diff
@@ -0,0 +1,31 @@
+%div
+  %b
+    = request_action_header(action, request_creator)
+  %hr
+- if action[:type] == :maintenance_incident && action[:releaseproject]
+  %i
+    Release in #{action[:releaseproject]}
+- action[:sourcediff].to_a.each do |sourcediff|
```

Comment:
why `to_a`?

Reply:
It was before (in the old UI) wrapped in a *.present? check, which I didn't like...

```ruby
- if action[:sourcediff].present?
  - action[:sourcediff].each do |sourcediff|
```

I changed it now to `(action[:sourcediff] || []).each`. Hope you can live with it?:-)